src/public/index.html

```diff
@@ -77,6 +77,7 @@ <h6 class="status-code status-code-5xx">5xx</h6>
   </div>
 </div>
 <script>
+  var port = {{port}};
```

Comment:
I would normally save this info inside a `data-*` attribute and read it inside the script w/ out polluting the global scope but ... not a major one